.travis.yml

```diff
@@ -1,18 +1,11 @@
 sudo: false
 language: go
 go:
-  - 1.5.4
-  - 1.6.3
-  - 1.7
-  - tip
+  - 1.7.x
```

Comment:
Now that 1.8 is out, this is done in 1ebf55f82387495e2e461928e4d8e248c4cd19cd (see line below).